docs/variables/downtime.md

```diff
@@ -0,0 +1,39 @@
+---
+id: downtime
+---
+
+# $(downtime)
+
+Returns the time that has passed since a user's stream has gone offline.
+
+#### Parameters
+
+This variable takes **one** *optional* parameter that is a Twitch username of who to fetch the downtime of. Defaults to the sender's username if not provided.
+
+#### Example Output
+
+* `$(downtime)`
+
+    ```
+    3 hours and 25 minutes
+    ```
+
+* `$(downtime fossabot)`
+
+    ```
+    12 hours and 2 minutes
+    ```
+
+#### Error Output
+
+* In case a users's stream is currently live, returns the following:
+
+    ```
+    [Error: Stream is live.]
+    ```
+
+* In case a users's stream has never been live before, returns the following:
```

Comment:
This is only if they've never been live *with fossabot*, Fossabot won't have any historical data to query, as it stores every game a user plays, and their streams.

Reply:
Yeah you're right, I should've made that more clear.

While I'm at it I'll also fix `users's` and change the whole thing to:

`In case a user's stream has never been live while Fossabot was in their chat, returns the following:`